test/stack_mitosis/policy_test.clj

```diff
@@ -95,4 +108,36 @@
            (sut/allow [:DeleteDBInstance]
                       (mapv fake-arn ["old-staging-replica" "old-staging"]))]}
          (sut/from-plan (example-instances)
-                        (plan/replace-tree (example-instances) "production" "staging")))))
+                        (plan/replace-tree (example-instances) "production" :none "staging"))))
+  (is (= {:Version "2012-10-17"
+          :Statement
+          [(sut/allow [:DescribeDBInstances :ListTagsForResource]
+                      ["arn:aws:rds:*:*:db:*"])
+           (sut/allow [:RestoreDBInstanceFromDBSnapshot]
+                      ["arn:aws:rds:*:*:og:*"
+                       "arn:aws:rds:*:*:pg:*"
+                       "arn:aws:rds:*:*:subgrp:*"
+                       (fake-snapshot-arn "*")
+                       (fake-arn "temp-staging")])
+           (sut/allow [:AddTagsToResource]
+                      (mapv fake-arn ["temp-staging" "temp-staging-replica"]))
+           (sut/allow [:DescribeDBSnapshots] ["*"])
+           (sut/allow [:ModifyDBInstance]
+                      (into ["arn:aws:rds:*:*:og:*"
+                             "arn:aws:rds:*:*:pg:*"
+                             "arn:aws:rds:*:*:secgrp:*"
+                             "arn:aws:rds:*:*:subgrp:*"]
+                            (mapv fake-arn ["temp-staging" "temp-staging-replica"
+                                            "staging-replica" "old-staging-replica"
+                                            "staging" "old-staging"])))
+           (sut/allow [:RebootDBInstance]
+                      (mapv fake-arn ["temp-staging" "temp-staging-replica" "staging-replica" "staging"]))
+           (sut/allow [:CreateDBInstanceReadReplica]
+                      (into ["arn:aws:rds:*:*:og:*"
+                             "arn:aws:rds:*:*:pg:*"
+                             "arn:aws:rds:*:*:subgrp:*"]
+                            (mapv fake-arn ["temp-staging" "temp-staging-replica"])))
+           (sut/allow [:DeleteDBInstance]
+                      (mapv fake-arn ["old-staging-replica" "old-staging"]))]}
+         (sut/from-plan (example-instances)
+                        (dbg (plan/replace-tree (example-instances) "production" "snapshot-id" "staging"))))))
```

Comment:
Looks like a stray `dbg` was left in the output but that method was removed? Had to remove this in order to run it through CI.